static/js/services/delete-docs.js

```diff
@@ -75,22 +108,33 @@ var _ = require('underscore');
         docs.forEach(function(doc) {
           doc._deleted = true;
         });
+        var parents = [];
         return $q.all(docs.map(function(doc) {
```

Comment:
This should be a `forEach`, not a `map`.

Reply:
I didn't write this but I think `map` is correct as far as I can tell, in order to generate an array of promises. Can you elaborate on why it should be `forEach`?

Reply:
Oh sorry you're right, ignore me!